config/locales/nl.yml

```diff
@@ -3,7 +3,7 @@ nl:
   sign_in: Inloggen
   locale_name: "Nederlands"
   sign_up: Aanmelden
-  footer_about: "RubyGems.org is de gem hosting service van de Ruby community. Publiceer and installeer je gems direct. Gebruik de API om meer informatie van beschikbare gems te vinden. Wordt een deelnemer en verbeter de site met jouw aanpassingen."
```

Comment:
I dunno dutch, but I think this was correct before. see https://translate.google.com/#nl/pt/Word%20een%20deelnemer%20en%20verbeter